api/outdated/outdated/synchroniser.py

```diff
@@ -45,49 +51,49 @@ async def _get_dependencies(self):
-                },
-                json={"query": q},
-            ) as response:
-                json = await response.json()
-                if json.get("message") == "Bad credentials":
-                    raise ValueError("API Token is not set")  # pragma: no cover
-                elif json.get("errors") and json["errors"][0]["message"] == "timedout":
-                    return await self._get_dependencies()  # pragma: no cover
-                elif json.get("errors"):
-                    raise ValueError(json)  # pragma: no cover
-                headers = response.headers
-                if headers.get("X-RateLimit-Remaining") == "0":  # pragma: no cover
-                    t = (
-                        int(headers.get("X-RateLimit-Reset"))
-                        - int(datetime.utcnow().timestamp())
-                    ) / 1000
-                    print(f"Rate limit exceeded. Sleeping for {t} seconds.")
-                    await sleep(t)
-                    return await self._get_dependencies()
-
-                lockfiles = []
-                lockfile_tasks = []
-                for lockfile in json["data"]["repository"]["dependencyGraphManifests"][
-                    "nodes"
-                ]:
-                    if basename(lockfile["blobPath"]) in LOCK_FILES:
-                        url = f"https://raw.githubusercontent.com/{lockfile['blobPath'].replace(f'blob/', f'')}"
-                        lockfile_tasks.append(session.get(url))
-                for lockfile_task in await gather(*lockfile_tasks):
-                    lockfiles.append(
-                        {
-                            "name": basename(str(lockfile_task.url)),
-                            "data": await lockfile_task.text(),
-                        }
-                    )
-
-                return await LockFileParser(lockfiles).parse()
+        async with ClientSession() as session, session.post(
+            "https://api.github.com/graphql",
+            headers={
+                "Authorization": f"Bearer {settings.GITHUB_API_TOKEN}",
+                "Accept": "application/vnd.github.hawkgirl-preview+json",
+            },
+            json={"query": q},
+        ) as response:
+            json = await response.json()
+            if json.get("message") == "Bad credentials":  # pragma: no cover
+                msg = "API Token is not set"
+                raise ValueError(msg)
+            if json.get("errors") and json["errors"][0]["message"] == "timedout":
+                return await self._get_dependencies()  # pragma: no cover
+            if json.get("errors"):
+                raise ValueError(json)  # pragma: no cover
+            headers = response.headers
+            if headers.get("X-RateLimit-Remaining") == "0":  # pragma: no cover
+                t = (
+                    int(headers.get("X-RateLimit-Reset"))
+                    - int(datetime.utcnow().timestamp())  # noqa: DTZ003
```

Comment:
This can be done without the `noqa`: `timezone.now().timestamp()` or set the timezone manually: `datetime.datetime.now(tz=)`.

Reply:
done here https://github.com/adfinis/Outdated/blob/a2f1613b561615e1c6cd31bd49622b93c80238e5/api/outdated/outdated/synchroniser.py#L72-L77